.github/workflows/actions.yml

```diff
@@ -793,3 +793,42 @@ jobs:
         with:
           name: binaries-${{ github.job }}
           path: pfx/**/*
+
+  android:
+    needs: [ what-to-make ]
+    runs-on: ubuntu-22.04
+    env:
+      VCPKG_DEFAULT_TRIPLET: x64-android
+    steps:
+    - name: Get Source
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+    
+    - name: Setup Gradle
+      uses: gradle/gradle-build-action@v2
+      with:
+        gradle-version: 7.6
+
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v3
+
+    - name: Install NDK
+      run: sdkmanager "ndk;26.1.10909125"
+
+    - name: Setup vcpkg
+      uses: lukka/run-vcpkg@v11
+      with:
+        vcpkgGitCommitId: c8696863d371ab7f46e213d8f5ca923c4aef2a00
+
+    - name: Build Transmission
+      working-directory: ./android
+      run: |
+        vcpkg install openssl curl --triplet x64-android
```

Comment:
Please split this step in two so that dependencies installation is a separate step. Also, passing `--triplet` seems unnecessary since you're already setting the `VCPKG_DEFAULT_TRIPLET` environment variable.